components/services/bat_ledger/bat_ledger_impl.cc

```diff
@@ -887,6 +887,19 @@ void BatLedgerImpl::DisconnectWallet(
                 _1));
 }
 
+void BatLedgerImpl::DisconnectExternalWallets(
+    DisconnectExternalWalletsCallback callback) {
+  auto weak_self = AsWeakPtr();
+  auto shared_callback =
```

Comment:
`std::function<>` is so awesome! 💔

Reply:
We should probably knock [this](https://github.com/brave/brave-browser/issues/18560) one out ASAP, so that we don't have to do such things. 🙂